### PR TITLE
Added agents-bulk-update-api

### PR DESCRIPTION
**Pull request by GaneshSPatil**

### CODE REVIEW

File: source/includes/agents/_60-bulk-delete.md.erb [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+Content-Type: application/vnd.go.cd.v2+json; charset=utf-8
+```
+
+```json
+{
+  "message": "Deleted 2 agent(s)."
+}
+```
+
+Deletes a group of agents.
```
Comment (ketan) [inline source/includes/agents/_60-bulk-delete.md.erb]: `list` not `group`

File: source/includes/agents/_60-bulk-delete.md.erb [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+<aside class="notice">
+  You must first <a href='#bulk-update-agents'>disable the agents</a> and ensure that their
+  <a href='#get-all-agents'>status</a> is not <code>Building</code>, before attempting to deleting them.
```
Comment (ketan) [inline source/includes/agents/_60-bulk-delete.md.erb]: Did you mean to say `#the-agent-object` instead of `#get-all-agents`

Comment (ketan) [inline source/includes/agents/_60-bulk-delete.md.erb]: The note on delete agent looks better — https://api.go.cd/current/#delete-an-agent

Comment (GaneshSPatil) [inline source/includes/agents/_60-bulk-delete.md.erb]: `#the-agent-object` will give the status of only one agent, whereas we need the status of all the agents that needs to be deleted so i used `#get-all-agents`

File: source/includes/agents/_50-bulk-update.md.erb [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+                "add": ["Dev", "Test"],
+                "remove": ["Production"]
+              },
+              "resources": {
+                "add": ["Linux", "Firefox"],
+                "remove": ["Crome"]
+              }
+            },
+            "agent_config_state" : "enabled"
+          }'
```
Comment (ketan) [inline source/includes/agents/_50-bulk-update.md.erb]: Could you add a quick section about what these attributes mean? Something along the lines of https://api.go.cd/current/#update-an-agent

**Pull request merged**